src/theme/Navbar/MobileSidebar/PrimaryMenu/index.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { useThemeConfig, ErrorCauseBoundary } from '@docusaurus/theme-common';
+import { splitNavbarItems, useNavbarMobileSidebar } from '@docusaurus/theme-common/internal';
+import NavbarItem from '@theme/NavbarItem';
+import './primary-menu.scss';
+import { Button } from '@deriv/ui';
+import {
+  LabelPairedGlobeCaptionRegularIcon,
+  StandaloneChevronLeftRegularIcon,
+} from '@deriv/quill-icons';
+
+import config from '@generated/docusaurus.config'; // Adjust the path as needed
+
+export function useNavbarItems() {
+  return useThemeConfig().navbar.items;
+}
+
+export default function CustomMobileSidebar() {
+  const [languageSidebarVisible, setLanguageSidebarVisible] = React.useState(false);
+  const mobileSidebar = useNavbarMobileSidebar();
+  const items = useNavbarItems();
+  const [leftItems] = splitNavbarItems(items);
+
+  React.useEffect(() => {
+    if (!mobileSidebar?.shown) {
+      setLanguageSidebarVisible(false);
+    }
+  }, [mobileSidebar]);
+
+  const toggleLanguageSidebar = () => {
+    setLanguageSidebarVisible(!languageSidebarVisible);
+  };
+
+  // Get the available locales from Docusaurus config
+  const { locales } = config.i18n;
+
+  return (
+    <React.Fragment>
+      <div>
+        {leftItems.map((item, i) => (
+          <ErrorCauseBoundary
+            key={i}
+            onError={(error) =>
+              new Error(
+                `A theme navbar item failed to render.
+            Please double-check the following navbar item (themeConfig.navbar.items) of your Docusaurus config:
+            ${JSON.stringify(item, null, 2)}`,
+              )
+            }
+          >
+            <NavbarItem
+              {...item}
+              onClick={() => {
+                mobileSidebar.toggle();
+              }}
+            />
+          </ErrorCauseBoundary>
+        ))}
+      </div>
+      <div className='navbar__item navbar__link' onClick={toggleLanguageSidebar}>
+        <LabelPairedGlobeCaptionRegularIcon /> EN
```

Comment:
@agrim-deriv Why 'EN' is fixed here? Shouldn't it change dynamically based on the chosen language?

Reply:
Fixed this @shafin-deriv